app.py

```
@@ -11,13 +11,34 @@
-RESULTS_PATH = 'results'
-JSON_RESULTS_PATH = 'results/json'
-INPUT_FILE = 'ticker_list.txt'
+def load_config(config_file='config.txt'):
+    """Lädt Konfiguration aus TXT-Datei"""
+    config = {}
+    try:
+        with open(config_file, 'r', encoding='utf-8') as f:
+            for line in f:
+                line = line.strip()
+                if line and not line.startswith('#'):
+                    if '=' in line:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)